### DISCUSSION TITLE
Translation of “VCs” in Italian

### DISCUSSION THREAD

**Discussion by enricomiletto** [e-id-admin/governance-sounding-board#34] [General]
Hello everyone,
I was reading the different translation of the draft for the new E-ID law (https://www.bj.admin.ch/bj/de/home/aktuell/mm.msg-id-89515.html, no English version) and I think the italian translation is not correct.
The "elektronischen Nachweisen", which (if I understand correctly how this stuff works) refers to the verifiable credentials and which are described in sections 3,4 and 5 of the draft, are translated to "mezzi di autenticazione elettronici", which means "means of electronic authentication" (imo not a good description for VCs).

I understand that in theory the term “authentication” can also refer to “the process of verifying the authenticity of something”, like a certificate for example, but in the context of computer security and digital identities I’ve always seen it used to describe user authentication, which is not the goal of VCs (in the general case).

I think that the most intuitive and straightforward naming would be to call VCs “certificati elettronici” (lit. electronic certificates), but I see how it could lead to confusion with the concept of digital certificates, which ofc have a very well defined meaning in computer security.
So maybe a literal translation from either the German version, “prove elettroniche” or the French version “moyens de preuve électroniques” —> “mezzi di prova elettronici” would be a good solution.

These are just my two cents. I have no knowledge about how to correctly write laws and maybe someone already has though a lot about this and realized that “mezzi di autenticazione elettronici” is the best way to name VCs in this legal context.

What do you think?

Enrico

**Comment by christianheimann** [comment]
Dear Enrico, even if it took a while to clarify, we let you know below the answer from our translator:

Ecco le ragioni della scelta terminologica:

1.	avendo utilizzato nel titolo dell’avamprogetto “mezzo d’identificazione” per Identitätsnachweis, ho mantenuto il termine “mezzo” (Nachweis) anche in queste sezioni per rispettare la coerenza terminologia richiesta dal testo normativo. Intitolare le sezioni 3 e 4 semplicemente “Altri mezzi elettronici” / “Utilizzo di mezzi elettronici” è totalmente fuorviante, quindi ho optato per un termine di per sé molto ampio ma sicuramente inclusivo rispetto a quanto si intende nello specifico e che, nei limiti del possibile, è un (buon) sinonimo di “certificato” (come sottolineato dal signor Enrico, utilizzare “certificato” nel testo sarebbe stato ambiguo visto che l’espressione torna in un altro contesto), autenticazione secondo il Treccani ([autenticazióne in Vocabolario - Treccani](https://www.treccani.it/vocabolario/autenticazione)) è un’ottima variante
2.	L’espressione “mezzi di autenticazione” (con la stessa imprecisione) ritorna con il medesimo significato anche nel regolamento europeo (n. 910/2014) di cui tiene conto l’avamprogetto in oggetto.
3.	Ho scartato a priori la nozione di “mezzi di prova” in quanto fortemente connotata (mezzi di prova nel procedimento giudiziario), ho scartato anche il termine “prove” in quanto troppo generico. 
4.	La coerenza terminologica ha le sue regole e i suoi vincoli… ovviamente, in un campo nuovo come quello dell’identificazione elettronica, tutto è perfettibile =)

Ringrazio il signor Enrico per il contributo, terrò/terremo conto delle sue osservazioni nella fase successiva dell’iter legislativo, spero che arrivino altri input linguistici magari dai partecipanti italofoni alla consultazione e spero inoltre che le mie spiegazioni siano sufficienti a giustificare scelte terminologiche non facili in un ambito giuridico nuovo e piuttosto tecnico.
